astropy/stats/jackknife.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from scipy.special import erfinv
```

Comment:
Re-reading the coding guidelines, yes, it would be better to move this line back to the function that uses it as it is the only function that uses it.   Sorry about this @mirca.  It can be moved back into the function without any wrapper around it.